ts/Series/Sankey/SankeySeries.ts

```diff
@@ -1258,16 +1269,50 @@ export default SankeySeries;
  * @type {string}
  * @product highcharts
  *//**
+ * This option is deprecated, use
+ * {@link Highcharts.SankeyNodeObject#offsetHorizontal} and
+ * {@link Highcharts.SankeyNodeObject#offsetVertical}  instead.
+ *
  * The vertical offset of a node in terms of weight. Positive values shift the
  * node downwards, negative shift it upwards.
  *
+ * If a percantage string is given, the node is offset by the percentage of the
+ * node size plus `nodePadding`.
+ *
  * @see {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/plotoptions/sankey-node-column/|Highcharts-Demo:}
  *         Specified node offset
  *
+ * @deprecated
  * @name Highcharts.SankeyNodeObject#offset
- * @type {number}
+ * @type {number|string}
  * @default 0
  * @since 6.0.5
+ *//**
+ * The horizontal offset of a node. Positive values shift the node right,
+ * negative shift it left.
```

Comment:
"Positive velues shift" or "Positive value shifts"?

Reply:
A copy-paste based on the previous `nodes.offset` description. There is no `Positive value shifts` that I can find in the file (and it probably should be `A positive value shifts`). However, I'm for the currently used wording. @TorsteinHonsi ?

Reply:
I think it's ok